.github/workflows/cypress.yml

```diff
@@ -5,13 +5,14 @@ on: [push]
 jobs:
   cypress:
     name: Run checks on ${{ matrix.browser }}
-    runs-on: [self-hosted, "${{ matrix.browser }}"]
+    runs-on: [self-hosted, '${{ matrix.browser }}']
```

Comment:
To jakaś różnica? Pytam bo raz przejebałem ze 3h przez to, że nie wiedziałem, że w ogóle trzeba jakieś dać.

Reply:
dla `'` nie musisz escapować znaków, dla `"` znaki specjalne musza być poprzedzone `\`, jak nie dasz `'` ani `"` to w przypadku gdy string zaczyna się jakimś specjalnym znakiem to nie zostanie to potraktowane jako string

Reply:
a w sumie to nie pamiętam żebym robił tą zmianę, możliwe że linter (?)